pixi.toml

```
@@ -214,6 +214,18 @@ py-docs-serve = "mkdocs serve -f rerun_py/mkdocs.yml -w rerun_py"
 # Lint markdown
 mdlint = "python scripts/ci/mdlint.py"
 
+[target.win-64.tasks]
+# Builds a debug mode Python wheel for install in an environment.
+py-wheel = { cmd = "cp target/debug/rerun.exe rerun_py/rerun_sdk/rerun_cli/rerun.exe && pixi run maturin build --manifest-path rerun_py/Cargo.toml --no-default-features --features pypi", depends_on = [
+  "py-build",
```

```suggestion
  "rerun-build-web-cli",
```
This doesn't actually depend on py-build. Also pypi activates web feature, so we need to build the web stuff.

---

We rather need both, right?

---

Not as far as I'm aware. 

`maturin dev` and `maturin build` are independent commands. They both cause the bindings to get build but then package the artifacts in different ways. 

`py-build` should really be called `py-dev`